invision-app/app/components/movieDetails/movieDetails.js

```diff
@@ -56,8 +57,16 @@ MovieDetails.propTypes = {
       genres: PropTypes.string.isRequired,
       runtime: PropTypes.oneOfType([PropTypes.string.isRequired, PropTypes.number.isRequired])
     }),
-    displayMainpage: PropTypes.func.isRequired
+    displayMainPage: PropTypes.func.isRequired
   };
   
 
-export default MovieDetails;
\ No newline at end of file
+  const mapStateToProps = (state) => {
+    return {
+      dmovie: state.movies.movie
+    };
+  };
+  
+
+export default connect(mapStateToProps, null)(MovieDetails);
```

Comment:
Instead of **connect** HOC you can use **useSelector** Redux hook to get movies from the store - [documentation](https://react-redux.js.org/api/hooks#useselector)